lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.html

```diff
@@ -19,7 +19,7 @@
     </button>
 
     <mat-menu #filter="matMenu"
-              class="adf-filter-menu"
+              class="adf-filter-menu adf-search-filter-menu"
```

Comment:
Yes, it would, but the css specificity from angular material, makes me do this that way as they do the same thing sometimes. I couldn't have found a better solution for that

Reply:
We can incorporate the style we need to add in the normal style flow, having multiple styles is something we avoided as it created a lot of confusion long ago, if this component need that style please add it to the relative one